.travis.yml

```diff
@@ -3,12 +3,11 @@ sudo: false
 language: python
 
 python:
-  - "2.7"
-  - "3.5"
+  - "3.6"
 
 install:
   # update periodically - saves downloading if version is the same
-  - if [[ "${TRAVIS_PYTHON_VERSION}" == "2.7" ]]; then
+  - if [[ "${TRAVIS_PYTHON_VERSION}" == "3.6" ]]; then
       wget http://repo.continuum.io/miniconda/Miniconda-3.8.3-Linux-x86_64.sh -O miniconda.sh;
```

Comment:
You're downloading miniconda for python 2 here.

You can remove the if statement and only download Miniconda3